src/ProjectTemplates/Web.ProjectTemplates/content/WebApi-CSharp/Startup.cs

```diff
@@ -9,15 +9,9 @@
 #endif
 using Microsoft.AspNetCore.Mvc;
 #if (OrganizationalAuth || IndividualB2CAuth)
+using Microsoft.AspNetCore.Authentication.JwtBearer;
```

Comment:
We try to follow alphabetical order in the using statements. In this case let's swap line 12 and line 13. Also double check the imports of the other files.

Reply:
nit: &hellip; but place System `using`s first

Reply:
Done;